src/init.ts

```diff
@@ -215,7 +215,9 @@ export async function init(options: Options): Promise<boolean> {
 
   // Run `npm install` after initial setup so `npm run check` works right away.
   if (!options.dryRun) {
-    cp.spawnSync('npm', ['install'], {stdio: 'inherit'});
+    // --ignore-scripts so that compilation doesn't happen because there's no
+    // source files yet.
+    cp.spawnSync('npm', ['install', '--ignore-scripts'], {stdio: 'inherit'});
```

Comment:
If someone has a `postinstall` script, I don't think that will get invoked either.  However, I expect this use case is probably not too common and, if needed, someone can always run `npm install` manually if needed.

Reply:
Yeah, most of the cases `gts init` is run from an empty directory. Even though someone had a `postinstall` script, they would know what they are doing and be able to figure things out.